astroquery/esasky/core.py

```diff
@@ -161,7 +161,7 @@ def query_object_catalogs(self, position, catalogs=__ALL_STRING,
         query_object_catalogs("265.05, 69.0", ["Gaia DR1 TGA", "HSC"])
         """
         return self.query_region_catalogs(position, self.__ZERO_ARCMIN_STRING, 
-                                          row_limit, catalogs, 
+                                          catalogs, row_limit, 
```

Comment:
Yup! I definitely see the benefit now :)

Btw, I am going to make a few more changes, so you don't need to merge yet.